tests/test_cityTemporalTiler.py

```diff
@@ -22,7 +22,7 @@ class Test_Tile(unittest.TestCase):
     def test_temporal(self):
         city_temp_tiler = CityTemporalTiler()
         output_dir = Path("tests/city_temporal_tiler_test_data/junk/temporal")
-        city_temp_tiler.args = Namespace(obj=None, loa=None, lod1=False, crs_in='EPSG:3946', crs_out='EPSG:3946', offset=[0, 0, 0], with_texture=False, output_dir=output_dir)
+        city_temp_tiler.args = Namespace(obj=None, loa=None, lod1=False, crs_in='EPSG:3946', crs_out='EPSG:3946', offset=[0, 0, 0], with_texture=False, output_dir=output_dir, split_surfaces=False)
```

Comment:
Could this 'EPSG:3946' default be another configurable "constant" ? (refer above)

Reply:
It will be done later (issue)[https://github.com/VCityTeam/py3dtilers/issues/81]